### DISCUSSION TITLE
Cat pics go here

### DISCUSSION THREAD

**Discussion by maltheism** [aces/Loris#7214] [Show and tell]
![file-20200803-24-50u91u](https://user-images.githubusercontent.com/16293415/102246327-a3f62480-3ecc-11eb-9b6e-a88631275d5a.jpg)

**Comment by maltheism** [comment]
![EfS0uLFX0AAl1dW](https://user-images.githubusercontent.com/16293415/102392859-9eb8d880-3fa5-11eb-949c-cc1f50a4ed92.jpg)

**Comment by maltheism** [comment]
![8bb95db3161128e930936bd9c64b26b8](https://user-images.githubusercontent.com/16293415/102558399-567be200-409b-11eb-824f-581feb52689d.jpg)

**Comment by maltheism** [comment]
![unnamed](https://user-images.githubusercontent.com/16293415/102662393-bf6f6280-414c-11eb-8754-918bd0e7a1d9.jpg)

**Comment by maltheism** [comment]
![holiday-mode-engaged](https://user-images.githubusercontent.com/16293415/102922747-87be2e80-445c-11eb-8910-d7b35eb67edb.jpg)

**Comment by maltheism** [comment]
![let_me_in](https://user-images.githubusercontent.com/16293415/103558125-2f3a5880-4e82-11eb-8ea5-a1c21da1b94b.gif)

**Comment by ridz1208** [comment]
@maltheism when is the next pic coming ?

**Comment by maltheism** [comment]
![01-cat-stretching-feet](https://user-images.githubusercontent.com/16293415/115418055-279b8700-a1c7-11eb-8483-1b297a5ff06d.gif)
@ridz1208

**Comment by maltheism** [comment]
![SickDistantLemming-size_restricted](https://user-images.githubusercontent.com/16293415/116878439-5c5b0580-abed-11eb-90d7-5c6f86c014bb.gif)

**Comment by maltheism** [comment]
![99767](https://user-images.githubusercontent.com/16293415/118513874-167c6200-b702-11eb-910a-c0fd3f16b4a5.jpg)

**Comment by maltheism** [comment]
![p22kori34du61](https://user-images.githubusercontent.com/16293415/124765826-5db5f080-df04-11eb-960c-af74f2e4a5a1.jpg)

**Comment by maltheism** [comment]
![cat-hello](https://user-images.githubusercontent.com/16293415/133111659-0ce7281b-76b2-4653-a21d-b5b9d804046b.gif)

**Comment by maltheism** [comment]
![landlord](https://user-images.githubusercontent.com/16293415/136056053-1679c3a1-1d4d-42d5-a304-585fcf0c3308.jpg)

**Comment by maltheism** [comment]
![cat and box](https://user-images.githubusercontent.com/16293415/139304635-b98124c2-341d-4be2-aef5-157c0d70e55b.gif)

**Comment by maltheism** [comment]
![cat-ducks-bathtub-cat](https://user-images.githubusercontent.com/16293415/142647316-a8e8e0eb-11ea-46b4-bd03-108978d93fa5.gif)

**Comment by maltheism** [comment]
![cat-squish](https://user-images.githubusercontent.com/16293415/143297600-0e9b661e-65f9-4e13-8d05-5345b46385ac.gif)

**Comment by maltheism** [comment]
![64453e60b392fc04a5415ba1014f07c5](https://user-images.githubusercontent.com/16293415/145891166-303d8daa-4641-4667-9e10-c46b26381b75.jpg)

**Comment by maltheism** [comment]
![ezgif-5-4f80eb95dc](https://user-images.githubusercontent.com/16293415/148268325-1fb5eafd-6231-497a-a3cb-1c84cb20f0ae.gif)

**Comment by maltheism** [comment]
![hug-love](https://user-images.githubusercontent.com/16293415/148865351-47ab2c09-bd1c-4432-a7a3-e43f354eb7e4.gif)

**Comment by ridz1208** [comment]
https://youtu.be/kEPfM3jSoBw

Just gonna leave this here

**Comment by maltheism** [comment]
![FgaJWfiWYAAd3QT jpeg](https://user-images.githubusercontent.com/16293415/199316675-16595d0e-05d4-4350-b4c4-5f05e26d79ff.jpg)

**Comment by maltheism** [comment]
![1nyfbxbwt6x91](https://user-images.githubusercontent.com/16293415/199317637-9b15012b-1fea-45a5-905a-bf34536fd9c1.jpg)

**Comment by maltheism** [comment]
![cat](https://user-images.githubusercontent.com/16293415/208966475-1158af06-2638-45e8-9721-dce738515913.gif)
